sdk/communication/communication-phone-numbers/src/models.ts

```diff
@@ -81,7 +88,12 @@ export interface ListSipRoutesOptions extends OperationOptions {}
 /**
  * Additional options that can be passed to list SIP trunks.
  */
-export interface ListSipTrunksOptions extends OperationOptions {}
+export interface ListSipTrunksOptions extends SipRoutingGetOptionalParams {}
```

Comment:
Feedback from review:
Instead of extending SipRoutingGetOptionalParams, we should keep the options as:
"ListSipTrunksOptions extends OperationOptions", but include property: "includeHealth?: boolean" instead, so the interface will look like this:

export interface ListSipTrunksOptions extends OperationOptions {
    includeHealth?: boolean;
}

Also, map the boolean property appropriately on the ExpandEnum behind the scenes (don't expose ExpandEnum anymore).

Reply:
Updated.